src/main/java/uk/gov/hmcts/reform/coh/controller/RelistingController.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.reform.coh.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.coh.controller.onlinehearing.Relisting;
+import uk.gov.hmcts.reform.coh.domain.OnlineHearing;
+import uk.gov.hmcts.reform.coh.domain.RelistingState;
+import uk.gov.hmcts.reform.coh.service.OnlineHearingService;
+
+import java.util.Optional;
+import java.util.UUID;
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping(
+    value = "/continuous-online-hearings/{onlineHearingId}/relist",
+    produces = MediaType.APPLICATION_JSON_VALUE
+)
```

Comment:
Missing Swagger docs

Reply:
Thanks, I'll add these in the upcoming commits.